openlibrary/core/yearly_reading_goals.py

```diff
@@ -52,6 +68,22 @@ def has_reached_goal(cls, username: str, year: int) -> bool:
         else:
             return results[0]['current'] >= results[0]['target']
 
+    @classmethod
+    def total_yearly_reading_goals(cls, since: date | None = None) -> int:
+        """Returns a Storage object of <Storage {'count': int}> where 'count' specifies the
+        number reading goals updated. `since` may be used
+        to limit the result to those reading goals updated since a specific
+        date. Any python datetime.date type should work.
+        :param since: returns all reading goals after date
+        """
+        oldb = db.get_db()
+
+        query = f"SELECT count(*) from {cls.TABLENAME}"
+        if since:
+            query += " WHERE updated >= $since"
+        results = list(oldb.query(query, vars={'since': since}))
+        return results[0]
```

Comment:
```suggestion
        results = oldb.query(query, vars={'since': since})
        return results[0]['count'] if results else 0
```
Just return an `int` here.  This follows the same pattern used in [`Booknotes.total_booknotes`](https://github.com/internetarchive/openlibrary/blob/672c5eddb38914a42f9f98418c19e898d83a1627/openlibrary/core/booknotes.py#L28-L34).